controls/postgres_spec.rb

```diff
@@ -38,10 +38,16 @@
   impact 1.0
   title 'Postgresql should be running'
   desc 'Postgresql should be running.'
-  describe service(postgres.service) do
-    it { should be_installed }
-    it { should be_running }
-    it { should be_enabled }
+  if os[:name] == 'ubuntu'
+    describe command('/etc/init.d/postgresql status') do
```

Comment:
Does this still exist on Ubuntu 22.04? Ubuntu uses systemd..

Reply:
Yes. I've tested on both my local machine and also your docker images for molecule.

Reply:
I also have some problems with using `if` here. What about we execute all `describe` blocks always and group them into a `describe.one`  block (https://docs.chef.io/inspec/profiles/controls/#check-if-at-least-one-condition-passes)

Reply:
I agree with both comments and starting to fix it.

Reply:
> I'm not really a fan of this solution. We depend on a file (that's not really in use anyway) and its output.. What do you think about doing it like in another baseline?
> 
> ```
> # set OS-dependent filenames and paths
> case os[:family]
> when 'ubuntu', 'debian'
>   process_name = 'postgresql'
> when 'redhat', 'fedora'
>   process_name = 'postgres'
> when 'suse'
>   process_name = 'postgresql'
> end
> ```
> 
> Note that these are probably not the correct names..

Is this example from MySQL baseline? If yes, there is no control for "mysql-conf-01" there. Can we remove this control for PostgreSQL, too? It seems the "postgres-03" also checks the state of the Postgres daemon.

Reply:
Yes, we dropped it here: https://github.com/dev-sec/mysql-baseline/pull/78

So I think, we should delete the test here, too.

@schurzi, thoughts?

Reply:
sounds reasonable. Let's do this

Reply:
Removed in 17eb756e6c0838d84d974cd32fb599d875582543.

Reply:
Removed in 17eb756e6c0838d84d974cd32fb599d875582543.